pep-0680.rst

```diff
@@ -356,22 +361,14 @@ Other considered names include:
   ``marshal``, ``html`` etc. would not be included in the namespace.
 
 
-TODO: Random things
+Previous Discussion
 ===================
 
-Previous discussion:
-
 * https://bugs.python.org/issue40059
-* https://mail.python.org/archives/list/python-ideas@python.org/thread/IWJ3I32A4TY6CIVQ6ONPEBPWP4TOV2V7/
 * https://mail.python.org/pipermail/python-dev/2019-May/157405.html
-* https://github.com/hukkin/tomli/issues/141
+* https://mail.python.org/archives/list/python-ideas@python.org/thread/IWJ3I32A4TY6CIVQ6ONPEBPWP4TOV2V7/
 * https://discuss.python.org/t/adopting-recommending-a-toml-parser/4068/84
-
-Useful https://grep.app searches (note, ignore vendored):
-
-* toml.load[s] usage https://grep.app/search?q=toml.load&filter[lang][0]=Python
-* toml.dump[s] usage https://grep.app/search?q=toml.dump&filter[lang][0]=Python
-* TomlEncoder subclasses https://grep.app/search?q=TomlEncoder%29%3A&filter[lang][0]=Python
```

Comment:
These were indeed very useful; instead of removing them completely, could you link them inline where relevant (discussion of write support, and in the differences from `toml` appendix)?

Reply:
A previous version of this PEP had a lot more text on a) reasons to include a write API, b) an appendix with discussion on the design space, but encukou removed this in https://github.com/hauntsaninja/peps/pull/3/files . 

https://github.com/hauntsaninja/peps/blob/9d497a22a755479eef02ee406458bf93e5766127/pep-9999.rst#appendix-b-designing-a-write-api
https://github.com/hauntsaninja/peps/blob/9d497a22a755479eef02ee406458bf93e5766127/pep-9999.rst#including-an-api-for-writing-toml

If we were to include these grep.app links in the PEP, it would probably make sense to restore at least some of that text. However, I wouldn't want to do that without encukou's approval, since he removed the discussion originally.

Maybe let's address this in a follow up, and not block the copyediting changes in this + your future PR on that.

Reply:
Hmm, thanks. While potentially useful, I suppose if a write API is considered out of scope, the PEP doesn't need to be belabored with lengthy material about it (though personally, I find much of the rationale compelling), so I can understand why Petr might have made the change.

In any case, I see this as rather orthogonal to inline-linking the links where they are relevant, as there are still sections (just not long ones) that refer to them. In particular, I suggest linking `significant fraction` in Appendix A - 1. to the `dump` link and `users` to the `load` link, and linking `two instances` (assuming that's the relevant reference) under Appendix A - 6. to the `TomlEncoder` link. I suggest also linking `one instance` under Appendix A - 5. to the relevant grep.app link.

Or, if you prefer, we can go ahead with this, and I can re-add them in my copyediting suggestions PR...up to you.

Reply:
Okay, yes. I think I focussed a little too hard on your suggestion of linking to this in "discussion of write support", but it definitely makes sense to cite these in the appendix. I've added links to justify use of "significant fraction".

I don't have clean grep.app links for the more niche stuff, I just went through hits for a couple search terms. I could link straight to Github for the one-two instance things, if you think that's worth it.

Reply:
Looks good; thanks @hauntsaninja !